agent/src/beerocks/slave/platform_manager/platform_manager_thread.cpp

```diff
@@ -2129,7 +2125,12 @@ bool main_thread::init_arp_monitor()
         m_uiArpMonIP   = info.ipa.s_addr;
         m_uiArpMonMask = info.nmask.s_addr;
 
-        if (bpl_arp_mon_start(&m_ctxArpMon, config.bridge_iface.c_str()) != 0) {
+        int ret = bpl_arp_mon_start(&m_ctxArpMon, config.bridge_iface.c_str());
+        if (ret < 0) {
+            if (ret == -BPL_ERR_OPERATION_NOT_SUPPORTED) {
+                LOG(INFO) << "Skip starting ARP monitor (not supported)";
+                return (true);
```

Comment:
I would add an explicit comment here to explain that it is not needed to initialize m_pArpRawSocket and m_pArpMonSocket and m_tpArpEntriesCleanup because they won't be used.

Also, maybe it would be better to not initialize m_uiArpMonIP and m_uiArpMonMask either in this case?

(nitpick)

Reply:
I'm not sure I understand why this comment is needed. I do however agree about moving the m_uiArpMonIP and m_uiArpMonMask initialization - and did that.